PSHTML/Functions/Public/sup.ps1

```diff
@@ -1,87 +1,80 @@
 Function SUP {
     <#
     .SYNOPSIS
-    Create a SUP title in an HTML document.
-
+        Create a SUP tag in an HTML document.
+    .DESCRIPTION
+        Create a SUP tag in an HTML document. 
```

Comment:
Some as for sub. You can find a more detailed description here https://www.w3schools.com/tags/tag_sup.asp